test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/ResponseTests.cs

```diff
@@ -2064,37 +2064,34 @@ public Task ResponseStatusCodeSetBeforeHttpContextDisposedRequestMalformedReadIg
         [MemberData(nameof(ConnectionAdapterData))]
         public async Task FailedWritesResultInAbortedRequest(ListenOptions listenOptions)
```

Comment:
Change the name of the test.

This should no longer be called FailedWrites*Result*InAbortedRequest. Now the test is written so that the request is *already* aborted before it simply verifies that calling WriteAsync(..., RequestAborted) throws. It's no longer a write that actually trips RequestAborted token like in Kestrel 1.x. Now LibuvConnection.OnRead() trips the RequestAborted token anytime the client closes the connection, not just for RSTs.